environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/request/azure/AzureResourceEncryptionParameters.java

```diff
@@ -0,0 +1,67 @@
+package com.sequenceiq.environment.api.v1.environment.model.request.azure;
+
+import com.sequenceiq.environment.api.doc.environment.EnvironmentModelDescription;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel(value = "AzureResourceEncryptionV1Parameters")
+public class AzureResourceEncryptionParameters {
+
+    @ApiModelProperty(EnvironmentModelDescription.KEY_VAULT_ID)
+    private String keyVaultId;
+
+    @ApiModelProperty(EnvironmentModelDescription.KEY_NAME)
+    private String keyName;
```

Comment:
same as above

Reply:
As mentioned above, I feel this shall be an URL for the specific key version.